src/components/Form/Comments/Comments.jsx

```diff
@@ -8,44 +8,49 @@ export default class Comments extends ValidationElement {
     super(props)
 
     this.state = {
-      value: this.props.value,
       visible: this.props.visible
     }
 
     this.toggle = this.toggle.bind(this)
     this.visible = this.visible.bind(this)
+    this.update = this.update.bind(this)
+    this.updateComments = this.updateComments.bind(this)
   }
 
-  toggle () {
-    let future = !this.visible()
-    let value = future ? this.state.value : ''
-    this.setState({ visible: future, value: value }, () => {
-      if (this.props.onUpdate) {
-        this.props.onUpdate(this.state.value)
-      }
+  update (queue) {
+    this.props.onUpdate({
```

Comment:
Correct, the `defaultProps` has the `onUpdate` function stubbed out so if it is now overridden it is still valid.

Is there a concern? Not really because the same could be said if someone were to set another default property with a different value type as well. This **should** be caught rather quickly through the various testing methods I would hope :smile: In addition this is a common pattern throughout the code base so hopefully a developer would pick up on this.

We could also use `propTypes` ([link](https://www.npmjs.com/package/prop-types)) to further guarantee it is the type we are expecting!